source/guide_prosody.rst

```diff
@@ -0,0 +1,309 @@
+.. highlight:: console
+.. author:: fapsi | fenchurch.space <fapsi@fenchurch.space>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/prosody.svg
+      :align: center
+
+#########
+Prosody
+#########
+
+Prosody_ is a modern XMPP communication server. It aims to be easy to set up and configure, and efficient with system resources. Additionally, for developers it aims to be easy to extend and give a flexible system on which to rapidly develop added functionality, or prototype new protocols.
+
+XMPP is an open and free alternative to commercial messaging and chat providers. Set it up for your company, organisation, or just your family and friends. You are in control, and your communication is private to you. Supporting a wide range of client software for desktop and mobile platforms, you can chat using Prosody from any device.
+
+Prosody is open source software under the permissive MIT/X11 license.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * supervisord_
+  * mysql_
+  * domains_
+  * ports_
+  * tls_
+
+Prerequisites
+=============
+Ports
+------------
+
+.. include:: includes/open-port.rst
+
+Altogether we need four open ports (referred to as: ``CLIENTPORT``, ``SERVERPORT`` and ``FILEUPLOADPORT`` ``FILETRANSFERPORT``) which should be public accessible.
+
+Domains, dns
+------------
+The following domains should be set up (to set up prosody for the domain ``stardust.space``):
+
+::
+ 
+ [isabell@stardust ~]$ uberspace web domain list
+ stardust.space
+ groupchat.stardust.space
+ [isabell@stardust ~]$
+
+Additionally to the dns_ A-records we need the following SRV-records: 
```

Comment:
Please add the `A` and `AAAA` records to the table below.

Reply:
Added!